`make reinstall` error

When running `make reinstall`, I get the following error:

```
Warning: The 'wp-app-manifest' plugin could not be found.
rm -f wp-app-manifest.zip
zip wp-app-manifest.zip -r wp-app-manifest/
	zip warning: name not matched: wp-app-manifest/

zip error: Nothing to do! (try: zip -r wp-app-manifest.zip . -i wp-app-manifest/)
make: *** [reinstall] Error 12
```

I think that's just an old relic that needs a name update.

--- a/Makefile
+++ b/Makefile
@@ -4,10 +4,10 @@ WP_CLI = tools/wp-cli.phar
 PHPUNIT = tools/phpunit.phar
 
 reinstall: $(WP_CLI)
-	$(WP_CLI) plugin uninstall --deactivate wp-app-manifest --path=$(WORDPRESS_PATH)
-	rm -f wp-app-manifest.zip
-	zip wp-app-manifest.zip -r wp-app-manifest/
-	$(WP_CLI) plugin install --activate wp-app-manifest.zip --path=$(WORDPRESS_PATH)
+	$(WP_CLI) plugin uninstall --deactivate wp-add-to-homescreen --path=$(WORDPRESS_PATH)
+	rm -f wp-add-to-homescreen.zip
+	zip wp-add-to-homescreen.zip -r wp-add-to-homescreen/
+	$(WP_CLI) plugin install --activate wp-add-to-homescreen.zip --path=$(WORDPRESS_PATH)
 
 test: $(PHPUNIT)
 	$(PHPUNIT)

